projects/cdk/directives/auto-focus/autofocus.options.ts

```diff
@@ -14,10 +14,12 @@ export interface TuiAutofocusHandler {
 
 export interface TuiAutofocusOptions {
     readonly delay: number;
+    readonly query: string;
 }
 
 export const TUI_AUTOFOCUS_DEFAULT_OPTIONS: TuiAutofocusOptions = {
     delay: NaN, // NaN = no delay/sync
+    query: 'input,textarea',
```

Comment:
Default should be `input, textarea, [contenteditable]`.

Reply:
Probably also `, select`

Reply:
fixed